UCLALibrary/voyager-archive
# SYS-983: Display purchase orders

Related to [SYS-983](https://jira.library.ucla.edu/browse/SYS-983)

This PR allows users to search by purchase order number, and displays data from the PO header and PO line items.
It also:
* Adds a new table `bib_title` which provides brief titles for display in PO lines (and invoice lines, later)
* Adds `bib_title.title` to `PoLineItemView` model and the underlying database view
* Fixes an error in `conversion_rate` in `PoLineItemView`
* Adds sample data allowing 1 line item to display for each sample purchase order
* Adds a `label` CSS class for styling table cells containing labels (separate from `<th>` table header cells)
* Updates python code in touched files via [Black](https://black.readthedocs.io/en/stable/) formatter

Testing: 
* Restart docker-compose system, if running
* Run: `sql_support/initialize_database.sh`
* Search for PO Number: `EAL518017` and `MUS522069`.  Each PO should display with one PO line, which is all I included in sample data.
